skimage/exposure/exposure.py

```diff
@@ -625,6 +625,11 @@ def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
     out : bool
         True when the image is determined to be low contrast.
 
+    Notes
+    -----
+    For boolean images, this function returns False only if all values are
+    the same (the method, threshold and percentile arguments are ignored).
```

Comment:
```suggestion
    the same (the method, threshold, and percentile arguments are ignored).
```
(yes, I love the Oxford comma)